devel/posix-macos-addons/Portfile

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+
+github.setup        stanislaw posix-macos-addons fb6766a9e76a15a71b1bed60b3ba7e229c7799c6
+version             20210922
+revision            0
+
+checksums           rmd160  6d592204047a80fca966e70c0ff6a85b1838a17b \
+                    sha256  5756d9af4e7fcfdaf81f88fbd6bfe317b160ecaf89f3cc8610a1d994be45a20f \
+                    size    271886
+
+categories          devel
+platforms           darwin
+license             Unknown
```

Comment:
Lack of a license might be a problem. I'd file a report on the original repo.

Reply:
@pmetzger very good point. Special in the case that it is a static linking library. Asked.